packages/tldraw/src/lib/ui/components/primitives/menus/TldrawUiMenuContext.tsx

```diff
@@ -29,7 +29,7 @@ export function useTldrawUiMenuContext() {
 
 /** @public */
 export interface TLUiMenuContextProviderProps {
-	type: TldrawUiMenuContextType
+	type: TLUiMenuContextType
```

Comment:
I guess it would make sense to unify the naming. `TLUiMenuContextProviderProps` for props and `TldrawUiMenuContextProvider` for the component. Feels weird.

Reply:
The naming is intentionally separated between just types (TLUi) and components (TldrawUi). There are just a ton of exports from tldraw, so this was my way of wrangling the proliferation / difference